documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,294 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+// {project-short} {project-z-version} is a maintenance release. There are no technical changes.
+
+This release has the following technical changes:
+
+.Migration from OpenStack moves to being a fully supported feature
+
+In this version of {project-short}, migration using OpenStack source providers graduated from a Technology Preview feature to a fully supported feature.
+
+.Integration of the create and update provider user interface
+
+The user interface of create and update providers now aligns with the look and feel of the {ocp} web console and displays up-to-date data.
+
+.Standalone UI
+
+The old UI of MTV 2.3 cannot run by setting `feature_ui: true` in ForkliftController anymore.
+
+.Improved errors logged in populator pods
+
+In earlier releases of {project-short} {project-version}, populator pods were always restarted on failure. This made it difficult to gather the logs from the failed pods. {project-short} 2.5.3 has a maximum number of three restarts of populator pods. On the third and final time, the populator pod remains in the fail status and its logs can then be easily gathered by must-gather and by forklift-controller to know this step has failed. link:https://issues.redhat.com/browse/MTV-818[(MTV-818)]
+
+.`VirtualMachineClusterFlavor`and `VirtualMachineClusterFlavor` CRDs replace OpenShift templates
+
+Virtual Machine Flavor API will replace OpenShift templates. MTV currently falls back to using OpenShift templates when a relevant preference is not available.
```

Comment:
* If you put this in codeblocks in the title then it has to be in codeblocks here
* There should be an article in front of `Virtual Machine Flavor API`. 
* `will replace` suggests this change is not implemented

```suggestion
The `Virtual Machine Flavor` API replaces OpenShift templates. MTV currently falls back to using OpenShift templates when a relevant preference is not available.
```